### DISCUSSION TITLE
Error reading QRCode

### DISCUSSION THREAD

**Discussion by imJohnYeager** [Auties00/WhatsappWeb4j#379] [Q&A]
Can anyone help me? when I read the QRCode, I get this error, I don't know what to do to find out the reason.

![image](https://github.com/Auties00/Cobalt/assets/130397626/5b827f58-501d-4490-b8e6-4b87e59507d0)

**Comment by bnagaraja** [comment]
@imJohnYeager Am also getting same error. Did you find any solution for the same

**Comment by bnagaraja** [comment]
is this group active? Because I am not getting any responses for user queries/doubts.
